app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,200 @@
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/btn_8"
+            app:layout_constraintTop_toBottomOf="@+id/btn_delete" />
+
+        <Button
+            android:id="@+id/btn_equal"
+            style="@style/btn_style"
+            android:text="@string/btn_text_equal"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/btn_point"
+            app:layout_constraintTop_toBottomOf="@+id/btn_3" />
+
+        <Button
+            android:id="@+id/btn_6"
+            style="@style/btn_style"
+            android:text="@string/btn_text_number_six"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/btn_5"
+            app:layout_constraintTop_toBottomOf="@+id/btn_9" />
+
+        <Button
+            android:id="@+id/btn_3"
+            style="@style/btn_style"
+            android:text="@string/btn_text_number_three"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/btn_2"
+            app:layout_constraintTop_toBottomOf="@+id/btn_6" />
+
+        <Button
+            android:id="@+id/btn_0"
+            style="@style/btn_style"
+            android:text="@string/btn_text_number_zero"
+            app:layout_constraintEnd_toStartOf="@id/btn_point"
+            app:layout_constraintHorizontal_chainStyle="packed"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/btn_1" />
+
+        <Button
+            android:id="@+id/btn_sub"
+            style="@style/btn_style"
+            android:layout_marginTop="@dimen/btn_margin_top"
+            android:text="@string/btn_text_sub"
+            app:layout_constraintEnd_toStartOf="@id/btn_multiply"
+            app:layout_constraintStart_toEndOf="@+id/btn_plus"
+            app:layout_constraintTop_toBottomOf="@+id/horizontal_scroll_text_view" />
+
+        <Button
+            android:id="@+id/btn_plus"
+            style="@style/btn_style"
+            android:layout_marginTop="@dimen/btn_margin_top"
+            android:text="@string/btn_text_plus"
+            app:layout_constraintEnd_toStartOf="@id/btn_sub"
+            app:layout_constraintHorizontal_chainStyle="packed"
+            app:layout_constraintStart_toStartOf="@+id/horizontal_scroll_text_view"
+            app:layout_constraintTop_toBottomOf="@+id/horizontal_scroll_text_view" />
+
+        <Button
+            android:id="@+id/btn_divide"
+            style="@style/btn_style"
+            android:layout_marginTop="@dimen/btn_margin_top"
+            android:text="@string/btn_text_divide"
+            app:layout_constraintEnd_toEndOf="@+id/horizontal_scroll_text_view"
+            app:layout_constraintStart_toEndOf="@+id/btn_multiply"
+            app:layout_constraintTop_toBottomOf="@+id/horizontal_scroll_text_view" />
+
+        <Button

```

Comment:
you can create a new style that inherits form btn_style that have the properties 

android:layout_marginTop="@dimen/btn_margin_top"
android:textAllCaps="false"

for the buttons like plus, multiply, delete, divide, sub and clear

Reply:
Done!